Rakefile

```
@@ -80,7 +82,8 @@ namespace :gems do
           attempts += 1
           sleep(2)
           begin
-            sh "gem push #{gem_path}"
+            sh "gem exec sigstore-cli:0.2.1 sign #{gem_path} --bundle #{gem_attestation_path}"
+            sh "gem push --attestation #{gem_attestation_path} #{gem_path}"
```

nit: can we reorder the args to
```suggestion
            sh "gem push #{gem_path} --attestation #{gem_attestation_path}"
```

---

Good idea, that also aligns with the [`gem push` docs](https://guides.rubygems.org/command-reference/#gem-push).